### PR TITLE
[Petition Field Changes ] Part 1 - Rename two petition snippet fields and add a new field

### PR DESCRIPTION
**Pull request by mmmavis**
# Description

Related PRs/issues: #10659

This is PR for part 1 of the work.

In this PR, 
- `requires_country_code` is renamed to `show_country_field`
- `requires_postal_code` is been renamed to `show_postal_code_field`
- a new boolean field `show_comment_field` is created
- value from the old `comment_requirements` field is mapped to the new `show_comment_field`
  - the current `none` value is mapped to`False`
  - the current `optional` value and `required` is mapped to `True`

@jhonatan-lopes Please let me know if I should squash the migrations or write any tests. Thanks!

# Checklist

**Tests**
- [ ] Is the code I'm adding covered by tests?

**Changes in Models:**
- [ ] Did I update or add new fake data?
- [ ] Did I squash my migration?
- [ ] Are my changes [backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/main/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations). If not, did I schedule a deploy with the rest of the team?

**Documentation:**
- [ ] Is my code documented?
- [ ] Did I update the READMEs or wagtail documentation?

**Merge Method**
**💡❗Remember to use squash merge when merging non-feature branches into `main`**

### CODE REVIEW

**Review by jhonatan-lopes — CHANGES_REQUESTED** [review]
Hi @mmmavis! It's all looking good, I just have some comments

File: network-api/networkapi/wagtailpages/migrations/0096_map_petition_comment_value.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+def map_old_to_new(apps, schema_editor):
+    Petition = apps.get_model("wagtailpages", "Petition")
+    for petition in Petition.objects.all():
+        if petition.comment_requirements == "none":
+            petition.show_comment_field = False
+        else:
+            petition.show_comment_field = True
+        petition.save()
```
[the remark this answers is not in the document]
Comment (mmmavis) [reply network-api/networkapi/wagtailpages/migrations/0096_map_petition_comment_value.py]: Thanks for the great suggestion! Didn't know this existed until now!

**Comment by mmmavis** [comment]
@jhonatan-lopes I've updated the PR according to your suggestions. 😄

**Review by jhonatan-lopes — APPROVED** [review]
Thanks @mmmavis! I've tested this locally and it's working well. I took the liberty to reorder the migrations to be able to merge it though

**Pull request merged**